benchmarks/styled-components/package.json

```diff
@@ -16,6 +18,8 @@
     "webpack-dev-server": "^3.4.1"
   },
   "dependencies": {
-    "styled-components": "^4.2.0"
+    "@xstyled/styled-components": "^1.0.3",
+    "styled-components": "^4.2.0",
+    "styled-system": "^4.2.4"
```

Comment:
Not part of this one.

Reply:
Same issue as [here](https://github.com/smooth-code/xstyled/pull/10#discussion_r288432213)